docs/deployment/considerations_production.md

```diff
@@ -0,0 +1,104 @@
+
+
+## Monitoring And Alerting
+
+**Consideration:** Proactively monitoring the IA environment is crucial for identifying performance issues, bottlenecks, and potential security threats.
+
+**Recommendation:** Leverage Azure Monitor to gain insights into the performance and health of your entire IA ecosystem. Set up custom alerts based on key performance indicators and metrics. Utilize Azure Log Analytics for in-depth analysis and troubleshooting. Integrate Azure Security Center to monitor and respond to security threats in real-time.
+Learn more [here](https://azure.microsoft.com/en-us/products/monitor/?ef_id=_k_2bb24bd93ec91aeba1fe2e4c90190298_k_&OCID=AIDcmm5edswduu_SEM__k_2bb24bd93ec91aeba1fe2e4c90190298_k_&msclkid=2bb24bd93ec91aeba1fe2e4c90190298).
+
+## Document Ingestion Limitations
+
+**Consideration:** This accelerator uses Azure Document Intelligence for PDF's and Unstructured.io for document processing. These tools have their own limitations when it comes to document sizes it can handle. The page limit and document size limits for PDFs on Document Intelligence can vary by subscription tier. Learn more about Document Intelligence [here](https://learn.microsoft.com/en-us/legal/cognitive-services/document-intelligence/characteristics-and-limitations).
+
+More info on unstructured.io for other supported document types can be found [here](https://unstructured-io.github.io/unstructured/introduction.html).
+
+**Recommendation:** If you see issues with the chunking or processing of very large documents, we recommend splitting those into a sensible format of multiple documents for ingestion.
+
+## Custom Branding
+
+If you would like to give custom branding to the Front End Web App interface, there are a couple ways this might be accomplished.
+
+### Configuring Existing UI
+
+**Consideration:** To give custom branding to the existing Front End Web App interface.
+This can be achieved by modifying the currently provided UI by making a couple of configuration changes.
+
+**Recommendation:** In the local.env used to deploy the solution, there exists a parameter called "APPLICATION_TITLE" that can be given any string value to change the title that appears on the title bar of the existing UI. You may also change the image files displayed by replacing them in the /app/frontend/src/assets folder. You may need to change the reference names in the code where it is refferencing if you change the file name of the referenced image file. The sky is the limit when it comes to modifying the existing front end source code for your own use cases.
+
+### Replacing Existing UI
+
+**Consideration:** Enabling token oauth implementation on the API could enable to have a different custom UI call the web app backend API.
```

Comment:
Need to reword this. Seems like an implementation suggestion versus a consideration, and the sentence structure is broken.